lib/droplet_kit/resources/container_registry_resource.rb

```diff
@@ -0,0 +1,26 @@
+module DropletKit
+    class ContainerRegistryResource < ResourceKit::Resource
+      include ErrorHandlingResourcable
+  
+      resources do
+        action :get, 'GET /v2/registry' do
+          handler(200) { |response| ContainerRegistryMapping.extract_single(response.body, :read) }
+        end
+  
+        action :create, 'POST /v2/registry' do
+          body { |object| ContainerRegistryMapping.representation_for(:create, object) }
+          handler(201) { |response, registry| ContainerRegistryMapping.extract_into_object(registry, response.body, :read) }
+          handler(422) { |response| ErrorMapping.fail_with(FailedCreate, response.body) }
+        end
+  
+        action :delete, 'DELETE /v2/registry' do
+          handler(202) { |response| true }
```

Comment:
Just to confirm – is this really a `202` over a `204`? There're only two delete actions in this API client that expect a `202` for deletes,
* `DELETE /v2/kubernetes/clusters/:id/node_pools/:pool_id`
* `DELETE /v2/kubernetes/clusters/:id`

Compared to all of the others that return a `204`.

Reply:
This should be 204, good catch 👍